src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int height;
+    private int base1;
+    private int base2;
+    private String color = getColor();
```

Comment:
Initializing 'color' with 'getColor()' is redundant since 'color' will be assigned in the constructor. You can remove this initialization.